src/arduino-nano-element.ts

```diff
@@ -0,0 +1,277 @@
+import { customElement, html, LitElement, property, svg } from 'lit-element';
+import { analog, ElementPin, i2c, spi, usart } from './pin';
+
+@customElement('wokwi-arduino-nano')
+export class ArduinoNanoElement extends LitElement {
+  @property() led13 = false;
+  @property() ledRX = false;
+  @property() ledTX = false;
+  @property() ledPower = false;
+  @property() resetButton = false;
```

Comment:
I think it'll make more sense to call it `resetPressed` (since it determines whether to draw the reset button in the pressed state or not, not whether to display it / turn it on/off)

Reply:
Good point. Naming things is hard. Fixed.